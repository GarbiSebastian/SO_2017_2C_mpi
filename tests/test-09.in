addAndInc una
addAndInc prueba
addAndInc con
addAndInc add
addAndInc and
addAndInc inc
addAndInc y
addAndInc con
addAndInc maximum
maximum
q
