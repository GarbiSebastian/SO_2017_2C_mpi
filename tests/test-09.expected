Comandos disponibles:
  load <arch_1> <arch_2> ... <arch_n>
  addAndInc <string>
  member <string>
  maximum
  q|quit
> Agregado: una
> Agregado: prueba
> Agregado: con
> Agregado: add
> Agregado: and
> Agregado: inc
> Agregado: y
> Agregado: con
> Agregado: maximum
> El máximo es <con,2>
> Finalizamos ejecución
